source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -110,442 +110,528 @@ volume and a supporting
+
+          .. code-block:: shell
+
+             NAME                               TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
+             minio                              LoadBalancer   10.97.114.60     <pending>     443:30979/TCP    2d3h
+             minio-tenant-1-console             LoadBalancer   10.106.103.247   <pending>     9443:32095/TCP   2d3h
+             minio-tenant-1-hl                  ClusterIP      None             <none>        9000/TCP         2d3h
+             minio-tenant-1-log-hl-svc          ClusterIP      None             <none>        5432/TCP         2d3h
+             minio-tenant-1-log-search-api      ClusterIP      10.103.5.235     <none>        8080/TCP         2d3h
+             minio-tenant-1-prometheus-hl-svc   ClusterIP      None             <none>        9090/TCP         7h39m
+
+          - The ``minio`` service corresponds to the MinIO Tenant service. 
+            Applications should use this service for performing operations against the MinIO Tenant.
+ 
+          - The ``*-console`` service corresponds to the :minio-git:`MinIO Console <console>`. 
+            Administrators should use this service for accessing the MinIO Console and performing administrative operations on the MinIO Tenant.
+
+          The remaining services support Tenant operations and are not intended for consumption by users or administrators.
  
-- The ``*-console`` service corresponds to the :minio-git:`MinIO Console 
-  <console>`. Administrators should use this service for accessing the MinIO
-  Console and performing administrative operations on the MinIO Tenant.
+          By default each service is visible only within the Kubernetes cluster. 
+          Applications deployed inside the cluster can access the services using the ``CLUSTER-IP``. 
+
+          Applications external to the Kubernetes cluster can access the services using the ``EXTERNAL-IP``. 
+          This value is only populated for Kubernetes clusters configured for Ingress or a similar network access service. 
+          Kubernetes provides multiple options for configuring external access to services. 
+          See the Kubernetes documentation on 
+          :kube-docs:`Publishing Services (ServiceTypes) <concepts/services-networking/service/#publishing-services-service-types>` 
+          and :kube-docs:`Ingress <concepts/services-networking/ingress/>` 
+          for more complete information on configuring external access to services.
+
+
+
+   .. tab-item:: Command Line (CLI)
+
+      The :mc:`kubectl minio tenant create` command supports creating a MinIO Tenant in your Kubernetes cluster.
+      The command *requires* that the cluster have a functional MinIO Operator installation.
+
+      You can also use this procedure to generate a Kubernetes ``YAML``-formatted resource document for further customization.
+
+      Prerequisite
+        Initialize the MinIO Operator.
+
+      1) Create a Tenant with default options
```

Comment:
So this is an example of how the step-style doesn't really work. These aren't sequential steps - they are parallel options. You can do 1 or 2 or 3, all of which result in 4.

If this was a standalone procedure, *this* is where tabs would make a lot of sense.

Reply:
When it comes to the CLI, it's really one step: run the `kubectl minio tenant create` command with the required and desired optional flags.

Reworded the steps to reflect a decision process rather than an action process.